its/plugin/pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>org.sonarsource.html</groupId>
     <artifactId>html-its</artifactId>
-    <version>3.4.0-SNAPSHOT</version>
+    <version>3.5.0-SNAPSHOT</version>
```

Comment:
was it intended to increase also the plugin version in this PR?

Reply:
Oups, I will revert the commit.

Reply:
I will wait for it to be green again and then I'll approve it :)

Reply:
However, I just noticed, that the its/plugin pom version is not identical to the other pom versions. Should be fixed in another commit.

Reply:
Thanks, I will create a separate PR for this.

Reply:
It seems that it was decreased few minutes ago with this commit:
https://github.com/SonarSource/sonar-html/commit/0f76403603148d7a50037436b05bede658e24ba2

Reply:
Forget my last comment :D